modules/gdscript/gdscript.cpp

```diff
@@ -1921,8 +1943,9 @@ Variant GDScriptInstance::callp(const StringName &p_method, const Variant **p_ar
 	return Variant();
 }
 
+// WARNING: `_notification()` uses multi-level calls, it gets called at **all** levels.
+// This behavior is an exception and is described in the documentation.
```

Comment:
Not sure, if `WARNING:` in the comment is necessary.
The behavior is documented in the docs.

The [style-guide](https://docs.godotengine.org/en/latest/contributing/development/code_style_guidelines.html#comment-style-guide) mentions, that comments should rather tell, why something is implemented the way it is and not how, so I would rather go with something like:

```suggestion
// `_notification()` uses multi-level calls, to ensure, that all base classes receive notifications no matter if the function is overridden.
```